### ISSUE TITLE
A suggestion on stable training

### ISSUE THREAD

**Issue by shzygmyx** [www0wwwjs1/Matrix-Capsules-EM-Tensorflow#12]
Hi, I am the author of [pytorch implementation of Matrix Capsules.](https://github.com/shzygmyx/Matrix-Capsules-pytorch)
I find that you are facing the problem of unstable training, which did not happen in my project. 
I guess it may due to my use of clipping method, both in gradient and variable,for example, if you have a/b or log(b) in your computational graph, you may need to clamp b to be bigger than a small number like 0.01  for computational stability.)

**Comment by www0wwwjs1** [comment]
Thanks a lot for your suggestion. The stability will be further improved.

**Comment by ashleylid** [comment]
Hi, was wondering if you did this change?
